.travis.yml

```diff
@@ -3,5 +3,7 @@ language: elixir
 otp_release:
   - 18.1
 elixir:
-  - 1.2.4
+  - 1.2.6
+  - 1.3.0
+before_install: "mix local.rebar --force"
```

Comment:
Without it, the build stalls because mix prompts you to install rebar to continue compiling dependencies, but there is no one to confirm the prompt.

I had to do the same thing in Delorean to get the build on 1.3 to not stall at that stage.